### DISCUSSION TITLE
TFT_eSPI and STM32cube-framework

### DISCUSSION THREAD

**Discussion by Zelsof** [Bodmer/TFT_eSPI#2139] [Q&A - General]
Hi Bodmer,

tft_espi is perfectly running on arduino framework. Now I think about switching to the stm32cube-framework, since I have a project on STM32F446 where I want to use some STM32 specific "onboard tools". I am wondering if it is still possible to use your libary? If not, do you know any suitable libary (ILI9488) for that framework?

Thank you in advance!

**Comment by Bodmer** [comment]
I am not familiar with the stm32cube-framework. I will flag as help needed.

**Comment by Bodmer** [comment]
Looks like help is not coming.  Moving to discussions.
